local_planner/src/nodes/local_planner.cpp

```diff
@@ -74,15 +77,21 @@ void LocalPlanner::dynamicReconfigureSetParams(
 }
 
 void LocalPlanner::setVelocity() {
-  velocity_mod_ = Eigen::Vector3f(curr_vel_.twist.linear.x,
-    curr_vel_.twist.linear.y, curr_vel_.twist.linear.z).norm();
+  velocity_mod_ =
+      Eigen::Vector3f(curr_vel_.twist.linear.x, curr_vel_.twist.linear.y,
+                      curr_vel_.twist.linear.z)
+          .norm();
```

Comment:
This does look a bit weird from a formatting perspective. Is it supposed to be this way?

Reply:
This is just clang-format... We could manually do formatting, but once this is all Eigen vectors this will be a simple `velocity_mod_ = cur_vel.twist.linear.norm();` anyway.

Reply:
Ok, sure, that's what I suspected. I just find this particular line pretty hard to read in clang. But you are right with Eigen it is gonna change anyway

Reply:
Ok, fixed :)